content/overview/workflow/process.md

```diff
@@ -0,0 +1,209 @@
+---
+title: process
+id: process
+draft: false
+weight: 30
+author: "people"
+---
+
+This section is intended to get you comfortable processing images in the darkroom view using a _scene-referred_ workflow. You are advised to follow the guidelines verbatim up to the end of the [_image processing in 3 modules_](#image-processing-in-3-modules) section and then choose other areas to learn as-and-when you need to use those techniques in your images. 
+
+# getting started
+
+## take a well-exposed photograph
+
+Good image processing techniques start in the camera -- a well-exposed image (without blown highlights or heavily crushed blacks) will always make post-processing much more straightforward. Under- or over-exposure can be "fixed" by darktable to some extent but no software can recover information that is not present in the Raw image. Where possible, you are advised to use [exposure to-the-right (ETTR)](https://en.wikipedia.org/wiki/Exposing_to_the_right) techniques to maximize the amount of data available for processing. As a general rule of thumb, it is safe to under-expose all images by 0.5 to 1 EV (by reducing the ISO sensitivity if possible) even if the in-camera preview looks darker than expected (the preview is not the raw data).
+
+## scene-referred workflow: a new approach
+
+If you have used other Raw software in the past (or darktable prior to version 3.0) you may notice some significant differences from what you are used to -- darktable now uses a _scene-referred_ approach for most of its processing modules. This approach is used extensively in cinematography and is known to be much more robust than the traditional _display-referred_ approach.
```

Comment:
I'm trying to keep it basic in the initial sections. I'm aware that it's a little hand-wavy but I want to keep this to a "do as we say" section rather than get into a massive explanation like we want the users to make a choice. The detail here is more about flagging to users that it's different and they're not necessarily "experts" any more.